ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1336,11 +1342,15 @@ router bgp {{ router_bgp.as }}
 {%             if neighbor.route_map_in is arista.avd.defined %}
       neighbor {{ neighbor.ip_address }} route-map {{ neighbor.route_map_in }} in
 {%             endif %}
+{#             TODO: AVD5.0 remove tmp_vrf_neighbor_prefix_list_in and tmp_vrf_neighbor_prefix_list_out #}
```

Comment:
Let's log an issue

TODO: AVD5.0 remove tmp_vrf_neighbor_prefix_list_in and tmp_vrf_neighbor_prefix_list_out. Special temporary variables set under vrf neighbors but read under vrf address_family_ipv4. This is required to move the deprecated vrfs[].neighbors[].ip_prefix_list_in/out to vrfs[].address_family_ipv4.neighbors[].ip_prefix_list_in/out #}